src/auth/AuthSessionHandler.cc

```diff
@@ -38,14 +38,3 @@ AuthSessionHandler *get_auth_session_handler(CephContext *cct, int protocol, Cry
   }
   return NULL;
 }
-
-
-void AuthSessionHandler::print_auth_session_handler_stats() {
```

Comment:
yeah, this method is not used since it was introduced[0,1]. i wonder if we are going to remove it, shall we remove `messages_signed`, `signatures_checked`, `signatures_matched`, `signatures_failed`,`messages_encrypted`,`messages_decrypted` member variables as well?

---
[0] ebcad66ede
[1] ffb8c605a8ae99cfa58bcfb66a23b9fa3a4d5e6d